client/.eslintrc.js

```diff
@@ -48,5 +49,8 @@ module.exports = {
       'always',
       { markers: ['#', '/'], exceptions: ['-'] },
     ],
+    'react-hooks/exhaustive-deps': 'off',
```

Comment:
Really want to make this a warning/error. I know there were some objections to it but its a "standard" react rule we should follow and IMHO doesn't hurt to fix some false positives. (btw, I just fixed a bug I introduced in this PR which this rule would have caught :))

Reply:
No I meant 'react-hooks/exhaustive-deps' :-)

Definetly trying to avoid `any`. I found there are supprisingly many type issues when using it. For example, I tried to fix one any warning and then run into some type issues, I left it though cause it got quite complicated and the part which caused issues wasn't used...